Language_Parser/actionVerbs.py

```diff
@@ -165,6 +165,10 @@ def take(info):
         allItems = player.getInventory() + room.getAccessibleItems()
         item = getSteppedItemName(allItems, item)
 
+    # Counts number of Polaroids player has collected
+    if "polaroid" in item and item in room.getVisibleItems():
+        player.incPolaroids()
```

Comment:
How does this work if the player drops and picks up a polaroid? Can it double count the same polaroid?

Reply:
That's what the `item in room.getVisibleItems()` is for. It will only register when the polaroid is first picked up (from the Visible Items list). Dropped items go in Dropped Items list, so there's no overlap.

Reply:
Ahh gotcha! Smart!